schemas/ontouml2.schema.json

```diff
@@ -88,6 +88,44 @@
         },
         "isDerived": {
           "$ref": "#/definitions/isDerived"
+        },
+        "isExtensional": {
+          "description": "A boolean field that captures if a class decorated as «collective» is extensionally defined (i.e., it's parts cannot change).",
+          "$ref": "#/definitions/nullableBoolean"
+        },
+        "isPowertype": {
+          "description": "A boolean field that captures if a class decorated as «type» represents a powertype.",
+          "$ref": "#/definitions/nullableBoolean"
+        },
+        "order": {
+          "description": "A string field that captures the type-order of a class decorated as «type».",
+          "$ref": "#/definitions/nullableString"
+        },
+        "allowed": {
```

Comment:
What should we put in this field if the class is a datatype or an enumeration? Don't we need natures for these?

Reply:
I like your proposal to use `"abstract"` as the nature of datatypes and enumerations. I'm in favor of having a consistent representation for different types of elements as much as possible.